app/services/image_service.rb

```diff
@@ -0,0 +1,16 @@
+class ImageService
+  def conn
+    Faraday.new(
+      url: Rails.configuration.x.image_service_url
+    )
+  end
```

Comment:
using an env variable still always makes me think of how this could be done with inheritance or a module instead... but we only have a couple services so not worth the hassle